iconservice/icon_service_engine.py

```diff
@@ -92,6 +94,7 @@ def __init__(self) -> None:
             'icx_getTotalSupply': self._handle_icx_get_total_supply,
             'icx_call': self._handle_icx_call,
             'icx_sendTransaction': self._handle_icx_send_transaction,
+            'icx_estimateStep': self._process_transaction,
```

Comment:
Isn't dangerous that for estimating a step cost, we invoke a transaction processing ? 
Shouldn't it call a specific method for that, such as `self._estimate_step` instead ?

Reply:
I know about dangerous issue too.
but We need to support API instantly which can calculate step each process to users who use that Network in ICON.

Therefore we due to provide only node like citizen node until find fit way which can access that API and execute.

Reply:
It would be better for estimating a step cost to be called by the specific method such as self._handle_estimate_step as you said. It will be applied. Thanks 👍